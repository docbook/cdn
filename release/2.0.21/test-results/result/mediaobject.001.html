<html xmlns="http://www.w3.org/1999/xhtml" xmlns:p="http://www.w3.org/ns/xproc" p:style=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Results for mediaobject.001.xml</title><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/prism.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/db-prism.css" /><link href="../style/show-results.css" rel="stylesheet" type="text/css" /><meta name="deltaxml" content="true" /></head><body><h1>DocBook XSLT 2.0 Stylesheet output: mediaobject.001.xml</h1><table class="diffs"><tbody><tr><th>Expected (<span class="exdiff">10</span> differences)</th><th>Actual (<span class="adiff">10</span> differences)</th></tr><tr><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Article wrapper&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Article wrapper&lt;/h2&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">mediaobject</span>"</span> <span class="att add">id="<span class="att">R.1.2</span>"</span>&gt;
            <span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">longdesc-link</span>"</span> <span class="att del">id="<span class="att"></span>"</span>&gt;<span class="diff txtchg">[</span><span class="tag diff del">&lt;a <span class="att">class="dialog-link"</span> <span class="att">href="#longdesc-xxx"</span> <span class="att">title="long description link"</span>&gt;D&lt;/a&gt;</span><span class="diff txtchg">]</span>&lt;/div&gt;</span><span class="diff txtchg">
            </span><span class="tag diff del">&lt;table <span class="att">border="0"</span> <span class="att">cellpadding="0"</span> <span class="att">cellspacing="0"</span> <span class="att">summary="Manufactured viewport for HTML image"</span> <span class="att">width="540"</span>&gt;
               <span class="tag">&lt;tr&gt;
                  <span class="tag">&lt;td <span class="att">align="right"</span>&gt;
                     <span class="tag">&lt;img <span class="att">align="right"</span> <span class="att">alt="The DocBook: TDG Duck"</span> <span class="att">longdesc="ld-R.1.2.html"</span> <span class="att">src="../graphics/duck-small.png"</span> <span class="att">width="540"</span>/&gt;</span>
                  &lt;/td&gt;</span>
               &lt;/tr&gt;</span>
            &lt;/table&gt;</span><span class="diff txtchg">
            </span><span class="tag diff del">&lt;div <span class="att">class="caption"</span>&gt;
               <span class="tag">&lt;p&gt;The Duck&lt;/p&gt;</span>
            &lt;/div&gt;</span>
         &lt;/div&gt;</span>
         <span class="tag nodiff">&lt;div <span class="att">class="longdesc-list"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="dialog longdesc-hide"</span> <span class="att">id="longdesc-xxx"</span>&gt;
               <span class="tag">&lt;p&gt;The bird on the cover of <span class="tag">&lt;em <span class="att">class="citetitle"</span>&gt;DocBook: The Definitive
Guide&lt;/em&gt;</span> is a wood duck.  Often considered one of the most
beautiful ducks in North America, the mail wood duck has a metallic
purple and green head with white streaks extending from its bill
around the eyes and down to its blue and green, gold-flecked
wings. It has a white neck, chestnut-colored chest, a white or red
bill, and yellow-orange legs and feet. Females have more brown, gray,
and subdueed hues.
&lt;/p&gt;</span>
               <span class="tag">&lt;p&gt;Edie Freedman designed the cover of this book, using a
19th-century engraving from the Dover Pictorial Archive. The
cover layout was produced by Kathleen Wilson using QuarkXPress
3.32 and Adobe's ITC Garamond font. The inside layout was
designed by Alicia Cech, based on a series design by Nancy
Priest. The text was formatted from SGML into FrameMaker 5.5
with Jade, using a DSSSL conversion stylesheet written by
Chris Maden. The interior fonts are ITC Garamond Light,
Garamond Book, and ConstantWillison.
&lt;/p&gt;</span>
            &lt;/div&gt;</span>
         &lt;/div&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="http://code.jquery.com/jquery-1.6.4.min.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/annotation.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Article wrapper&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Article wrapper&lt;/h2&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">content</span>"</span> <span class="att del">id="<span class="att"></span>"</span>&gt;
            <span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">mediaobject</span>"</span> <span class="att add">id="<span class="att">R.1.2</span>"</span>&gt;<span class="diff txtchg">
               </span><span class="tag diff add">&lt;div <span class="att">class="longdesc-link"</span>&gt;[<span class="tag">&lt;a <span class="att">class="dialog-link"</span> <span class="att">href="#longdesc-xxx"</span> <span class="att">title="long description link"</span>&gt;D&lt;/a&gt;</span>]&lt;/div&gt;</span><span class="diff txtchg">
               </span><span class="tag diff add">&lt;table <span class="att">border="0"</span> <span class="att">cellpadding="0"</span> <span class="att">cellspacing="0"</span> <span class="att">summary="Manufactured viewport for HTML image"</span> <span class="att">width="540"</span>&gt;
                  <span class="tag">&lt;tr&gt;
                     <span class="tag">&lt;td <span class="att">align="right"</span>&gt;
                        <span class="tag">&lt;img <span class="att">align="right"</span> <span class="att">alt="The DocBook: TDG Duck"</span> <span class="att">longdesc="ld-R.1.2.html"</span> <span class="att">src="../graphics/duck-small.png"</span> <span class="att">width="540"</span>/&gt;</span>
                     &lt;/td&gt;</span>
                  &lt;/tr&gt;</span>
               &lt;/table&gt;</span><span class="diff txtchg">
               </span><span class="tag diff add">&lt;div <span class="att">class="caption"</span>&gt;
                  <span class="tag">&lt;p&gt;The Duck&lt;/p&gt;</span>
               &lt;/div&gt;</span><span class="diff txtchg">
            </span>&lt;/div&gt;</span>
         &lt;/div&gt;</span>
         <span class="tag nodiff">&lt;div <span class="att">class="longdesc-list"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="dialog longdesc-hide"</span> <span class="att">id="longdesc-xxx"</span>&gt;
               <span class="tag">&lt;p&gt;The bird on the cover of <span class="tag">&lt;em <span class="att">class="citetitle"</span>&gt;DocBook: The Definitive
Guide&lt;/em&gt;</span> is a wood duck.  Often considered one of the most
beautiful ducks in North America, the mail wood duck has a metallic
purple and green head with white streaks extending from its bill
around the eyes and down to its blue and green, gold-flecked
wings. It has a white neck, chestnut-colored chest, a white or red
bill, and yellow-orange legs and feet. Females have more brown, gray,
and subdueed hues.
&lt;/p&gt;</span>
               <span class="tag">&lt;p&gt;Edie Freedman designed the cover of this book, using a
19th-century engraving from the Dover Pictorial Archive. The
cover layout was produced by Kathleen Wilson using QuarkXPress
3.32 and Adobe's ITC Garamond font. The inside layout was
designed by Alicia Cech, based on a series design by Nancy
Priest. The text was formatted from SGML into FrameMaker 5.5
with Jade, using a DSSSL conversion stylesheet written by
Chris Maden. The interior fonts are ITC Garamond Light,
Garamond Book, and ConstantWillison.
&lt;/p&gt;</span>
            &lt;/div&gt;</span>
         &lt;/div&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="http://code.jquery.com/jquery-1.6.4.min.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/annotation.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td></tr><tr><td><em>The “current” CSS styling is applied
                  to the this entire document; the results in this
                  column may be anomalous. Also, image links are broken
                  in this column.</em></td><td> </td></tr><tr><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><div id="R.1.2" class="mediaobject"><div class="longdesc-link">[<a class="dialog-link" href="#longdesc-d24793e42" title="long description link">D</a>]</div><table border="0" summary="Manufactured viewport for HTML image" cellspacing="0" cellpadding="0" width="540"><tr><td align="right"><img src="../graphics/duck-small.png" align="right" width="540" alt="The DocBook: TDG Duck" longdesc="ld-R.1.2.html" /></td></tr></table><div class="caption"><p>The Duck</p></div></div><div class="longdesc-list"><div id="longdesc-d24793e42" class="dialog longdesc-hide"><p>The bird on the cover of <em class="citetitle">DocBook: The Definitive
Guide</em> is a wood duck.  Often considered one of the most
beautiful ducks in North America, the mail wood duck has a metallic
purple and green head with white streaks extending from its bill
around the eyes and down to its blue and green, gold-flecked
wings. It has a white neck, chestnut-colored chest, a white or red
bill, and yellow-orange legs and feet. Females have more brown, gray,
and subdueed hues.
</p><p>Edie Freedman designed the cover of this book, using a
19th-century engraving from the Dover Pictorial Archive. The
cover layout was produced by Kathleen Wilson using QuarkXPress
3.32 and Adobe's ITC Garamond font. The inside layout was
designed by Alicia Cech, based on a series design by Nancy
Priest. The text was formatted from SGML into FrameMaker 5.5
with Jade, using a DSSSL conversion stylesheet written by
Chris Maden. The interior fonts are ITC Garamond Light,
Garamond Book, and ConstantWillison.
</p></div></div></article><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script src="../../resources/js/prism.js"></script></td><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><div class="content"><div id="R.1.2" class="mediaobject"><div class="longdesc-link">[<a class="dialog-link" href="#longdesc-d55715e42" title="long description link">D</a>]</div><table border="0" summary="Manufactured viewport for HTML image" cellspacing="0" cellpadding="0" width="540"><tr><td align="right"><img src="../graphics/duck-small.png" align="right" width="540" alt="The DocBook: TDG Duck" longdesc="ld-R.1.2.html" /></td></tr></table><div class="caption"><p>The Duck</p></div></div></div><div class="longdesc-list"><div id="longdesc-d55715e42" class="dialog longdesc-hide"><p>The bird on the cover of <em class="citetitle">DocBook: The Definitive
Guide</em> is a wood duck.  Often considered one of the most
beautiful ducks in North America, the mail wood duck has a metallic
purple and green head with white streaks extending from its bill
around the eyes and down to its blue and green, gold-flecked
wings. It has a white neck, chestnut-colored chest, a white or red
bill, and yellow-orange legs and feet. Females have more brown, gray,
and subdueed hues.
</p><p>Edie Freedman designed the cover of this book, using a
19th-century engraving from the Dover Pictorial Archive. The
cover layout was produced by Kathleen Wilson using QuarkXPress
3.32 and Adobe's ITC Garamond font. The inside layout was
designed by Alicia Cech, based on a series design by Nancy
Priest. The text was formatted from SGML into FrameMaker 5.5
with Jade, using a DSSSL conversion stylesheet written by
Chris Maden. The interior fonts are ITC Garamond Light,
Garamond Book, and ConstantWillison.
</p></div></div></article><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script src="../../resources/js/prism.js"></script></td></tr></tbody></table><h2>XML source</h2><pre data-src="../src/mediaobject.001.xml"></pre></body><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script type="text/javascript" src="../../resources/js/prism.js"></script></html>