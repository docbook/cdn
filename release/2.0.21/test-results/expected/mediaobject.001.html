<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Article wrapper</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><div id="R.1.2" class="mediaobject"><div class="longdesc-link">[<a class="dialog-link" href="#longdesc-d24793e42" title="long description link">D</a>]</div><table border="0" summary="Manufactured viewport for HTML image" cellspacing="0" cellpadding="0" width="540"><tr><td align="right"><img src="../graphics/duck-small.png" align="right" width="540" alt="The DocBook: TDG Duck" longdesc="ld-R.1.2.html" /></td></tr></table><div class="caption"><p>The Duck</p></div></div><div class="longdesc-list"><div id="longdesc-d24793e42" class="dialog longdesc-hide"><p>The bird on the cover of <em class="citetitle">DocBook: The Definitive
Guide</em> is a wood duck.  Often considered one of the most
beautiful ducks in North America, the mail wood duck has a metallic
purple and green head with white streaks extending from its bill
around the eyes and down to its blue and green, gold-flecked
wings. It has a white neck, chestnut-colored chest, a white or red
bill, and yellow-orange legs and feet. Females have more brown, gray,
and subdueed hues.
</p><p>Edie Freedman designed the cover of this book, using a
19th-century engraving from the Dover Pictorial Archive. The
cover layout was produced by Kathleen Wilson using QuarkXPress
3.32 and Adobe's ITC Garamond font. The inside layout was
designed by Alicia Cech, based on a series design by Nancy
Priest. The text was formatted from SGML into FrameMaker 5.5
with Jade, using a DSSSL conversion stylesheet written by
Chris Maden. The interior fonts are ITC Garamond Light,
Garamond Book, and ConstantWillison.
</p></div></div></article><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script src="../../resources/js/prism.js"></script></body></html>